modules/light-clients/08-wasm/testing/simapp/app.go

```diff
@@ -488,7 +489,12 @@ func NewSimApp(
 		MemoryCacheSize:   uint32(math.Pow(2, 8)),
 		ContractDebugMode: false,
 	}
-	app.WasmClientKeeper = wasmkeeper.NewKeeperWithConfig(appCodec, keys[wasmtypes.StoreKey], authtypes.NewModuleAddress(govtypes.ModuleName).String(), wasmConfig)
+	if mockVM != nil {
+		// NOTE: mockVM is used for testing purposes only!
+		app.WasmClientKeeper = wasmkeeper.NewKeeperWithVM(appCodec, keys[wasmtypes.StoreKey], authtypes.NewModuleAddress(govtypes.ModuleName).String(), mockVM)
```

Comment:
this is a bit icky but I think I can live with it, we'll need this for the E2Es to work I guess!